src/node/db/PadManager.js

```diff
@@ -170,6 +171,11 @@ exports.sanitizePadId = async (padId) => {
     padId = padId.replace(from, to);
   }
 
+  // If caseInsensitivePadNames is enabled, we need to make sure the padId is lowercase
```

Comment:
This comment is unnecessary; the code is easy enough to read.

```suggestion
```

Reply:
applied in 68fa4f19c45f23c01c8f6f0c54793f78cd1ce930